### ISSUE TITLE
Pain on weight bearing

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11078]
**Preferred term label:**
Pain on weight bearing

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Many, including:
Hip Fracture
Tibial shaft fracture
Calcaneus fractures
Ankle sprain

**Your nano-attribution (ORCID)**
